hack/run-ci-e2e-test.sh

```diff
@@ -102,6 +102,11 @@ OSDK_WMCO_test $OSDK "-run=TestWMCO/operator_deployed_without_private_key_secret
 
 # Run the creation tests of the Windows VMs
 OSDK_WMCO_test $OSDK "-run=TestWMCO/create -v -timeout=90m -node-count=$NODE_COUNT --private-key-path=$KUBE_SSH_KEY_PATH $WMCO_PATH_OPTION"
+# Get logs for the creation tests
```

Comment:
Please add a `echo \n####### WMCO logs for creation tests #######\n` or something along those lines here. Finding it hard to figure out which pod logs belongs to which set of tests. Something similar would be needed before the other `get_WMCO_logs` also: `echo \n####### WMCO logs for upgrade and deletion tests #######\n`.